### ISSUE TITLE
show puppetfile subcommand table formatting is not aligned

### ISSUE THREAD

**Issue by TraGicCode** [dylanratcliffe/onceover#182]
When running `onceover show puppetfile` on mac the table is not formatted properly.  On Mac, Mine looks like so:

```
FULL NAME                 | CURRENT VERSION | LATEST VERSION | OUT OF DATE?   | ENDORSEMENT | SUPERSEDED BY
--------------------------|-----------------|----------------|----------------|-------------|---------------
httpsrewrite              | N/A             | N/A            | N/A            | N/A         | N/A
gcc-gcc_package           | N/A             | N/A            | N/A            | N/A         | N/A
octopus-tentacle          | N/A             | N/A            | N/A            | N/A         | N/A
gcc-externalfacts         | N/A             | N/A            | N/A            | N/A         | N/A
bryana-ec2tagfacts        | 0.3.0           | 0.3.0          | No    |             |
puppetlabs-acl            | 2.1.0           | 2.1.0          | No    | supported   |
puppetlabs-scheduled_task | 1.0.0           | 1.0.0          | No    | supported   |
puppet-iis                | 2.0.2           | 3.1.0          | Major |             | puppetlabs-iis
puppetlabs-java           | 2.4.0           | 3.2.0          | Major | supported   |
puppetlabs-registry       | 2.1.0           | 2.1.0          | No    | supported   |
binford2k-node_encrypt    | 0.3.3           | 0.3.3          | No    |             |
puppetlabs-apt            | 6.1.1           | 6.1.1          | No    | supported   |
opentable-iis_rewrite     | 1.0.0           | 1.0.0          | No    |             |
puppet-download_file      | 3.1.0           | 3.2.0          | Minor |             |
puppet-windowsfeature     | 2.1.0           | 3.2.1          | Major | approved    |
puppetlabs-powershell     | 2.1.5           | 2.1.5          | No    | supported   |
puppetlabs-reboot         | 2.0.0           | 2.0.0          | No    | supported   |
rtyler-jenkins            | 1.7.0           | 1.7.0          | No    | approved    |
puppet-archive            | 3.2.0           | 3.2.0          | No    | approved    |
puppetlabs-chocolatey     | 3.1.0           | 3.1.0          | No    | supported   |
puppetlabs-dsc            | 1.4.0           | 1.7.0          | Minor | supported   |
puppet-nodejs             | 6.0.0           | 6.0.0          | No    |             |
puppet-splunk             | 7.2.1           | 7.2.1          | No    | approved    |
puppetlabs-inifile        | 2.4.0           | 2.4.0          | No    | supported   |
puppetlabs-stdlib         | 5.1.0           | 5.1.0          | No    | supported   |
```

**Comment by dylanratcliffe** [comment]
Oh yeah what the hell? I wonder when that broke...

**Comment by dylanratcliffe** [comment]
This was broken by #178, need to investigate how to interact with [table_print](https://rubygems.org/gems/table_print) so that it prints correctly

**Comment by dylanratcliffe** [comment]
Fixed in #273
